.gitignore

```diff
@@ -3,3 +3,5 @@ npm-debug.log
 
 coverage/
 .nyc_output/
+
+.idea
```

Comment:
Eh...
https://piecioshka.pl/blog/2017/02/06/globalna-konfiguracja-gita.html

Reply:
fixed in https://github.com/piecioshka/xhr-sniffer/pull/2/commits/5eb001f398bffb39763a96f8e81f8c3fbd2a549e